components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -3866,11 +3895,20 @@ subroutine ma_resuspend_convproc(                           &
 !           end if
 
 ! cam5 approach
-            dcondt(la,k) = qdotac
-            dcondt(lc,k) = 0.0
-
-            dcondt_resusp(la,k) = (dcondt(la,k) - qdota)
-            dcondt_resusp(lc,k) = (dcondt(lc,k) - qdotc)
+!            dcondt(la,k) = qdotac
+!            dcondt(lc,k) = 0.0
+!
+!            dcondt_resusp(la,k) = (dcondt(la,k) - qdota)
+!            dcondt_resusp(lc,k) = (dcondt(lc,k) - qdotc)
```

Comment:
same as above, are these commented lines important, if git will keep the history of what the code used to look like previously?